cmd/commands/function.go

```diff
@@ -122,21 +80,22 @@ func FunctionCreate(fcTool *core.Client, invokers, buildpacks map[string]string)
 					if createFunctionOptions.Namespace != "" {
 						namespaceOption = fmt.Sprintf(" -n %s", createFunctionOptions.Namespace)
 					}
-					fmt.Fprintf(cmd.OutOrStdout(), "Issue `%s service status %s%s` to see the status of the function\n", env.Cli.Name, fnName, namespaceOption)
+					_, _ = fmt.Fprintf(cmd.OutOrStdout(), "Issue `%s service status %s%s` to see the status of the function\n", env.Cli.Name, fnName, namespaceOption)
 				}
 			}
 
 			return nil
 		},
 	}
 
-	LabelArgs(command, "INVOKER", "FUNCTION_NAME")
+	LabelArgs(command, "FUNCTION_NAME")
 
 	command.Flags().StringVarP(&createFunctionOptions.Namespace, "namespace", "n", "", "the `namespace` of the service")
 	command.Flags().BoolVarP(&createFunctionOptions.DryRun, "dry-run", "", false, dryRunUsage)
 	command.Flags().StringVar(&createFunctionOptions.Image, "image", "", "the name of the image to build; must be a writable `repository/image[:tag]` with credentials configured")
-	command.Flags().StringVar(&createFunctionOptions.InvokerURL, "invoker-url", "", "the path to a custom invoker url. Required if invoker is custom.")
 	command.MarkFlagRequired("image")
+	command.Flags().StringVar(&createFunctionOptions.Invoker, "invoker", "", "invoker runtime to override `language` detected by buildpack")
+	command.Flags().StringVar(&createFunctionOptions.BuildpackImage, "builder", builder, "the `repository/image[:tag]` coordinates of a custom buildpack builder [local builds only]")
```

Comment:
The builder is required either as a switch or as a default. In other words, if the default is blank, the switch is required